src/libraries/System.Runtime/tests/System.Runtime.Tests/System/Runtime/CompilerServices/ConditionalWeakTableTests.cs

```diff
@@ -185,6 +218,63 @@ public static void Concurrent_Add_Read_Remove_DifferentObjects()
             });
         }
 
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsThreadingSupported))]
+        public static void Concurrent_GetOrAdd_Add_Read_Remove_DifferentObjects()
+        {
+            var cwt = new ConditionalWeakTable<object, object>();
+            DateTime end = DateTime.UtcNow + TimeSpan.FromSeconds(0.25);
```

Comment:
It's not a big deal, but it looks like all of these .25 second tests are going to add several seconds to the execution of the test suite. Can we lower these values?

Reply:
Not sure why this value was chosen in the first place (the new tests just reused the same value that other existing tests were using), and if it was this much for a specific reason. Do you know who might know? Or, should we consider potentially lowering this in a follow up, since that value was already there before this PR?

Reply:
It's an arbitrary value that's only in three tests, for a total of < 1 second. These additions are quadrupling that.

Reply:
I lowered the value to 0.1 seconds for all new tests. Left it at 0.25 for the old tests to not reduce coverage there.